README.md

```diff
@@ -26,6 +26,22 @@ There are no dependencies.
 
 Below is a sample playbook:
```

Comment:
As we added a new sample, we might need to alter this like "Below are sample usages for playbook:"